server/migrations/20211110231126-add-channel-owners.js

```diff
@@ -0,0 +1,26 @@
+exports.up = async function (db) {
+  await db.runSql(`
```

Comment:
This migration does not seem complete enough, it needs to set proper values for channels that already exist. I realize we haven't shipped multiple channels in production yet, but we do have them locally, and getting our local channels into some state that won't exist on production is going to lead to a bunch of weird errors and misaligned expectations. You can probably use a similar query to the user leaving ones to find the right owner